c2rust-analyze/src/rewrite/mod.rs

```diff
@@ -62,6 +62,12 @@ pub enum Rewrite<S = Span> {
     Cast(Box<Rewrite>, String),
     /// The integer literal `0`.
     LitZero,
+    /// Cell::new
+    CellNew,
+    /// Cell::get
+    CellGet(Box<Rewrite>),
+    /// cell.set(x)
+    CellSet(Box<Rewrite>, Box<Rewrite>),
```

Comment:
It seems likely that we'll add more function/method-call rewrites in the future, so it might be worth generalizing these with something like `FunctionCall(String, Vec<Rewrite>)` for calling arbitrary functions (and similarly for methods).  This would be similar to `TyCtor` which can build arbitrary generic types.

Reply:
Does 957bbfee cover the extent of what you were thinking?